src/gateway/request.js

```diff
@@ -74,9 +70,8 @@ export const request = (
             return res;
         })
         .finally(() => {
-            traceSession.end({
-                retries: retryCount,
-            });
+            traceSession.addLabel("retries", retryCount);
```

Comment:
Changed this from metadata added to the log message into a label that will be included in the log message AND the trace.